docs/sites.yml

```diff
@@ -10500,6 +10500,19 @@
   featured: false
   main_url: https://sztuka-programowania.pl/
   url: https://sztuka-programowania.pl/
+- title: Rivers Casino
+  built_by: WILDLIFE.LA
+  built_by_url: https://www.wildlife.la
+  categories:
+    - Entertainment
+    - Food
+    - Blog
+    - Travel
+  description: >
+    Rivers Casino offers the very best in casinos, hotels, restaurants, concerts, and entertainment. Visit us in Des Plaines, IL, Philadelphia, PA, Pittsburgh, PA and Schenectady, NY.
+  featured: false
+  main_url: https://www.riverscasino.com
```

Comment:
Since these urls point to what isn't a Gatsby site, I think it could get removed in the future when we sweep through. Could they be updated to one of the 4 linked sites?

Reply:
done